### DISCUSSION TITLE
Uploaded proof has already been used previous

### DISCUSSION THREAD

**Discussion by Lee0JJ** [community/community#66265] [GitHub Education]
### Select Topic Area

Question

### Body

I used to have two accounts which are my account and my university account, I registered for the student developer pack using the university account and used my student ID as proof. However I found out that it would be inconvenient to use both accounts, so I deleted the university account and linked it to my account. Since the university account has been deleted, I assume that I have to apply for the student developer pack again, but the error message "Your uploaded proof has already been used previously" popped out.

Will the application proceed successfully after I delete the account and add it to my personal account?
If not, is there any solution for me to reapply for the student developer pack again?

**Comment by Itumadavid1** [comment]
Unfortunately, once you've used a specific document or proof for applying to the GitHub Student Developer Pack, you typically cannot reuse the same proof for a different account. Deleting the original account and linking it to your personal account doesn't reset the document verification status.

To resolve this issue, you should contact GitHub Support directly. They will be able to assist you in this specific situation and help you reapply for the Student Developer Pack with your personal account.

You can contact GitHub Support through their support portal or by emailing them at support@github.com. Explain your situation clearly, and they should be able to guide you through the process of reapplying or provide an alternative solution.

**Comment by Lee0JJ** [comment]
I have tried to email the customer support that you suggested but here's
the email that I receive from them. Looks I'm trapped in a loop...

We now require that new support requests be created using our Support
website:

https://support.github.com

Please use this website to search through resources that may help you find
the solutions you are looking for or connect with our GitHub experts.

Thank you,
GitHub Support



Ituma David I ***@***.***> 于 2023年9月7日周四 下午1:13写道：

**Reply by nabinyd** [reply]
is your problem solved or not? if yes How did you solve this issue? i am also stuck with same issue

**Reply by vanssssssss** [reply]
try to upload another picture. it worked for me

**Comment by Lee0JJ** [comment]
Yes it is solved

vanssssssss ***@***.***> 于 2024年3月2日周六 下午11:00写道：
